group_vars/all/public.yml

```diff
@@ -0,0 +1,5 @@
+tarsnap_version: 1.0.37
+tarsnap_cache: /var/tarsnap/cache
+tarsnap_cron_hour: "*"
+
+FORWARD_MAIL_RELAY_MAIL_TO: 'jacek@opencraft.com'
```

Comment:
Can you change the (public) default to a non-functioning addres, like `ops@example.com`, so that we don't get mail if others reuse our roles? Then override it in the private repo.